extensions/tldr/package.json

```
@@ -0,0 +1,43 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "tldr",
+  "title": "tldr",
+  "author": "pomdtr",
+  "license":"MIT",
+  "description": "Search tldr Pages",
+  "icon": "tldr.png",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Search TLDR Pages",
```

```suggestion
      "title": "Search Pages",
```

Redundant, already added to the subtitle.